packages/orchestration/src/types.ts

```diff
@@ -11,3 +11,20 @@ export type * from './orchestration-api.js';
 export type * from './exos/cosmos-interchain-service.js';
 export type * from './exos/chain-hub.js';
 export type * from './vat-orchestration.js';
+
+/**
+ * ({@link ZCF})-like tools for use in {@link OrchestrationFlow}s.
+ *
+ * @interface
```

Comment:
what does `@interface` do here?

Reply:
It makes tsdoc format it like an interface
 - https://github.com/Agoric/agoric-sdk/issues/9766

might as well use `export interface ...` though.